ext/opcache/jit/zend_jit_trace.c

```diff
@@ -6071,6 +6071,29 @@ static const void *zend_jit_trace(zend_jit_trace_rec *trace_buffer, uint32_t par
 							goto jit_failure;
 						}
 						goto done;
+					case ZEND_FETCH_STATIC_PROP_FUNC_ARG:
+						if (!JIT_G(current_frame)
+						 || !JIT_G(current_frame)->call
```

Comment:
These ``func`` check seem not necessary.
JIT/FFI patch is going to remove them, because it adds an ability to inline FFI calls implemented through trampolines (without known ``func``).
Here I didn't add the check in the first place.